### DISCUSSION TITLE
Load JS Library with WASM

### DISCUSSION THREAD

**Discussion by TonyK-FmGroup** [unoplatform/uno#8610] [Q&A]
Hello
I am trying to place my JS in a library and access this from c#.  I can execute JS as strings from c# but I'm struggling to access the JS when I place it in a library.

Question 1:
I've been following the tutorials and on this page:
https://platform.uno/docs/articles/interop/wasm-javascript-2.html
There is this link which is returning a 404: https://github.com/unoplatform/uno/tree/master/src/Uno.UI.Wasm   
Is this repo available? It may have the answer I need

Question 2:
I have my library in WasmScripts and added the line to the cs proj as seen here:

<img width="763" alt="image" src="https://user-images.githubusercontent.com/103487880/165071132-766ea60d-79c9-45da-8db4-6f10b4fdf48f.png">

I see the script has loaded in the browser but I've not found how to then include my libary in a script tag so that I can access the library.

<img width="898" alt="image" src="https://user-images.githubusercontent.com/103487880/165071341-80897970-fd6e-4091-a553-6ae9e6a24c69.png">

Maybe I missed something in the tutorial?

Edit:
This page  https://platform.uno/docs/articles/interop/wasm-javascript-1.html?
says that "they will be copied to the output folder and loaded automatically by the bootstrapper when the page loads." which I understood as meaning the js in my myapp.js would be loaded and therefor executed.

**Comment by TonyK-FmGroup** [comment]
This was my mistake, I had an error in the JS which I hadn't see in the console. So, myapp.js is loaded and executed, but we won't see it in the HEAD tag but will see it in the package.

**Reply by sasakrsmanovic** [reply]
Thanks for reporting the 404 , we will look for the missing link and replace it.

**Reply by sasakrsmanovic** [reply]
@TonyK-FmGroup  - we will update the link but the project has been moved here https://github.com/unoplatform/uno/blob/master/src/Uno.UI/Uno.UI.Wasm.csproj . Thanks @carldebilly  for helping locate it.
